meteor/sofie-server/data-access/repositories/mongo/mongo-entity-converter.ts

```diff
@@ -7,12 +7,24 @@ import { Timeline } from '../../../model/entities/timeline'
 import { Identifier } from '../../../model/interfaces/identifier'
 import { AdLibPiece } from '../../../model/entities/ad-lib-piece'
 import { PieceLifeSpan } from '../../../model/enums/piece-life-span'
+import { BasicRundown } from '../../../model/entities/basic-rundown'
 
 export interface MongoIdentifier {
 	_id: string
 	name: string
 }
 
+export interface MongoRundownPlaylist extends Document {
```

Comment:
What do you get from extending from Document? It seems unnecessary

Reply:
Nothing really, i have no idea why that was there :-D